css/templates/e-commerce/page.css

```diff
@@ -717,6 +728,33 @@ amp-carousel {
   border-top: 1px solid var(--color-separator);
 }
 
+.commerce-related-products amp-carousel {
+  padding-left: 40px;
+  padding-right: 40px;
+}
+
+.commerce-related-products .amp-carousel-button {
```

Comment:
if this applies to all your carousel buttons , its cool to do `.amp-carousel-button`